futures/examples/functional.rs

```diff
@@ -0,0 +1,57 @@
+#![feature(futures_api, async_await, await_macro)]
+#[macro_use] extern crate futures;
+
+use futures::{
+    channel::mpsc,
+    executor, //standard executors to provide a context for futures and streams
+    futures::ready,
+    StreamExt
```

Comment:
Usually we use the expanded style:

```rust
use futures::channel::mpsc;
use futures::executor::block_on;
use futures::future;
use futures::spawn;
use futures::stream::StreamExt;
```

(We use `{...}`, but only for imports from the same module and not nested)